eq-author/src/components/Badge/index.js

```diff
@@ -38,14 +38,14 @@ export const smallBadge = css`
 
 export const mediumBadge = css`
   border-radius: 0.7em;
-  color: white;
   line-height: 1;
   margin-left: auto;
   font-size: 0.9rem;
 `;
 
 const Badge = styled.span`
-  background-color: ${colors.red};
+  background-color: ${colors.red} !important;
```

Comment:
You are lucky the linter hasn't pulled you up on the !important

Reply:
That was on purpose; the badge should always have this style but I needed to do a ` > span` elsewhere, which effected this.

Reply:
Sometimes the linter doesn't allow. I love using !important but have had my knuckles wrapped